client/components/loading-placeholder/index.jsx

```diff
@@ -0,0 +1,35 @@
+/**
+ * External dependencies
+ */
+import classnames from 'classnames';
+import noop from 'lodash/utility/noop';
+import page from 'page';
+import React from 'react';
+
+/**
+ * Internal dependencies
+ */
+import HeaderCake from 'components/header-cake';
+import Main from 'components/main';
+
+const LoadingPlaceholder = React.createClass( {
```

Comment:
`LoadingPlaceholder` seems fine to me.We don't prefix components with `Page` so I don't think we should do it for this placeholder.